handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1351,20 +1351,20 @@ public final void setEnabledCipherSuites(String[] cipherSuites) {
                 return enabled.toArray(new String[1]);
             }
         }
-        if (isProtocolEnabled(opts, SSL.SSL_OP_NO_TLSv1, OpenSsl.PROTOCOL_TLS_V1)) {
-            enabled.add(OpenSsl.PROTOCOL_TLS_V1);
+        if (isProtocolEnabled(opts, SSL.SSL_OP_NO_TLSv1, SslUtils.PROTOCOL_TLS_V1)) {
+            enabled.add(SslUtils.PROTOCOL_TLS_V1);
```

Comment:
consider just adding a static import for these ... less verbose and they are used multiple times in this file.